src/pretty.rs

```diff
@@ -238,7 +238,7 @@ where
                         }, // be sure we have at least 1 `%` sign when an interpolation is present
                     )
                     .max()
-                    .unwrap();
+                    .unwrap_or(1);
```

Comment:
Can we get rid of the `else { 1 }` above if we use `1` as a default value here? But I think those kind of improvements, albeit small, would be better off as a separate PR.